src/components/ExperienceCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+const ExperienceCard = ({ role, company, details, id }) => {
+  return (
```

Comment:
can you make this function auto return? (remove the return and change the {} with ())

Reply:
done